### ISSUE TITLE
Import Repositories?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#179]
```
Is it possible to import saved repositories from another copy of svnx? For 
instance I just did a clean install from Snow Leopard to Lion and would like to 
have all of the repos I had prior to the install.
```

Original issue reported on code.google.com by `ja...@jasonwiener.com` on 18 Jul 2012 at 6:36

**Comment by GoogleCodeExporter** [comment]
```
If you retained your user folder then there should be nothing to do.
SvnX simply stores its settings in 
<your-home-folder>/Library/Preferences/com.lachoseinteractive.svnX.plist.
[It also maintains caches in ~/Library/Caches/com.lachoseinteractive.svnX/, but 
these will be rebuilt as needed.]

It’s possible to copy the svnX’s lists of working copies & repositories 
from one user to another by simply
copying the above mentioned Preferences file.  However, I get the impression 
that this is not your problem.
Do you still have svnX’s Preferences file?

This discussion would be best continued on 
<http://groups.google.com/group/svnx>.
```

Original comment by `chris...@gmail.com` on 18 Jul 2012 at 4:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `chris...@gmail.com` on 27 Jul 2012 at 3:22
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
same problem but your solution did not work, I  tried to copy the preference 
file from an installation to another (on different computers) and it does not 
work, because svnx on restart will apparently erase the preference file I copied

any help is appreciated
```

Original comment by `ori...@dis.uniroma1.it` on 18 Dec 2013 at 10:06

**Comment by GoogleCodeExporter** [comment]
```
SvnX does not/will not erase a prefs file.
However, if Cocoa cannot read a prefs file it will probably result in an 
app’s prefs returning to their defaults (which may ultimately replace the 
unreadable file).

When I wrote “simply copying the … file” the fact that you may also need 
to update its read/write privileges & ownership for the new user was implied.
Did you try that?
```

Original comment by `chris...@gmail.com` on 18 Dec 2013 at 4:10

**Comment by GoogleCodeExporter** [comment]
```
It turns out that the cache files will cause a problem. Three times now, I've 
wanted to move the repo list from an old machine to a new one. All three times, 
I'd already fired up svnX when I remembered this. In each case, I'd copy the 
pref file, launch svnX, then watch as it replaced the pref file with a default 
one. And yes, owner, group, and perms were the same. In all three cases, I 
resolved the problem by deleting the cache files and trying again.

So to summarize how to do this:

1 Delete the cache files. You can simply nuke the whole 
~/Library/Caches/com.lachoseinteractive.svnX folder.
2 Copy the pref file.
3 Enjoy.

Too late for the original poster (and for me, three times), but hopefully, this 
will help someone in the future.
```

Original comment by `bob.crys...@gmail.com` on 10 Jan 2015 at 12:11
